tests/Fixtures/tags/for/loop_in_else.test

```diff
@@ -0,0 +1,25 @@
+--TEST--
+"for" tag loop behavior in an else clause
+--TEMPLATE--
+{% for item in [] %}
+{% else %}
+{{ loop.previous is null ? 'OK' : 'KO' }}
+{{ loop.next is null ? 'OK' : 'KO' }}
+{{ loop.first ? 'OK' : 'KO' }}
+{{ loop.last ? 'OK' : 'KO' }}
+{{ loop.index0 is same as 0 ? 'OK' : 'KO' }}
+{{ loop.revindex0 is same as 0 ? 'OK' : 'KO' }}
+{{ loop.length is same as 0 ? 'OK' : 'KO' }}
+{{ loop.depth is same as 1 ? 'OK' : 'KO' }}
```

Comment:
Should be 0 maybe?

Reply:
to me, it is fine to be 1. This is the depth of the current loop (even if it was empty)